DEPRECATIONS.md

```diff
@@ -4,6 +4,10 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
+| `ChatClient#unMuteChannel(channelType, channelId)`<br/>*client* | 2021.03.15<br/>4.7.1  | 2021.04.15 ⌛ | 2021.05.15 ⌛ | Use the `unmuteChannel(channelType, channelId)` method instead |
```

Comment:
We could make these deprecation entries simpler if we removed the param lists and just mentioned the names of the methods